src/components/Layout.tsx

```diff
@@ -1,34 +1,40 @@
 import { Box, Container, Heading, HStack } from "@chakra-ui/react"
 import Head from "next/head"
 import Account from "components/web3Connection/Account"
-import { Token } from "temporaryData/types"
+import { useContext } from "react"
+import { CommunityContext } from "./community/Context"
 
 type Props = {
-  title: string
-  token: Token
+  title?: string
   bg?: string
   children: JSX.Element
 }
 
-const Layout = ({ title, token, bg = "white", children }: Props): JSX.Element => (
-  <>
-    <Head>
-      <title>{title}</title>
-      <meta property="og:title" content={title} />
-      {/* <link rel="icon" href="/favicon.ico" /> */}
-    </Head>
-    <Box bg={bg} minHeight="100vh">
-      <Container maxW="container.lg" py={24} px={10}>
-        <HStack justify="space-between" align="center" pb={16}>
-          <Heading size="2xl" fontFamily="display">
-            {title}
-          </Heading>
-          <Account token={token} />
-        </HStack>
-        {children}
-      </Container>
-    </Box>
-  </>
-)
+const makeTitle = (name: string) => `${name} community`
```

Comment:
This is specific to the community page. There may be other pages in the future that uses Layout, so just leave title as a required prop and pass the string from the community page. All changes in this file except the removal of the token can be reverted